### PR TITLE
Web feature/address payment api

### PR DESCRIPTION
**Pull request by ykaya932**
- Connected the address and payment page to the backend.
- Used `services.js` to get the base url

### CODE REVIEW

File: web/carousel/src/apis/Account/Requests.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+const getElements = (field, setState) => {
+  const config = {
+    headers: { Authorization: `Bearer ${TOKEN}` },
+  };
+
+  services
+    .get("customer/me", config)
+    .then((res) => {
+      console.log(res.data.data[field]);
```
Comment (afraarslan) [inline web/carousel/src/apis/Account/Requests.js]: we can delete these console logs since there are tons of them. :sweat_smile:

Comment (afraarslan) [reply web/carousel/src/apis/Account/Requests.js]: I realized that this console log gives an undefined error. Can you check if there is a mistake here?

**Review by afraarslan — CHANGES_REQUESTED** [review]
I think it needs some changes.

* Check the `PaymentInfo.js` screen and the `Request.js` file for undefined values.
* Delete console logs except for errors (Errors may help when debugging)
* Delete unused variables

File: web/carousel/src/apis/Account/Requests.js (hunk 1 above)
Comment (ykaya932) [reply web/carousel/src/apis/Account/Requests.js]: Sure, thanks for the review!

Comment (ykaya932) [reply web/carousel/src/apis/Account/Requests.js]: Thanks for pointing that error out. Now, I am making sure that at least the list is an empty list :+1:

**Review by ahmetyigitgedik — APPROVED** [review]
Thanks for your effort 💯

**Review by afraarslan — APPROVED** [review]
Problems are solved.Thanks  :robot:

**Pull request merged**